packages/extunix/extunix.0.3.1/opam

```diff
@@ -0,0 +1,68 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "Collection of thin bindings to various low-level system API"
+description: """
+Motto: "Be to Unix, what extlib is to stdlib"
+
+* Implement thin C bindings that directly map to underlying system API.
+* Provide common consistent ocaml interface: naming convention, exceptions.
+* Simple to build - no extra dependencies.
+"""
+maintainer: ["ygrek@autistici.org"]
+authors: [
+  "Andre Nathan"
+  "Antonin Décimo"
+  "Dmitry Grebeniuk"
+  "François Bobot"
+  "Gerd Stolpmann"
+  "Goswin von Brederlow"
+  "Joshua Smith"
+  "Kaustuv Chaudhuri"
+  "Markus W. Weissmann"
+  "Mehdi Dogguy"
+  "Niki Yoshiuchi"
+  "Pierre Chambart"
+  "Roman Vorobets"
+  "Stéphane Glondu"
+  "Sylvain Le Gall"
+  "ygrek"
+  "Zhenya Lykhovyd"
+]
+license: "LGPL-2.1-only with OCaml-LGPL-linking-exception"
+tags: ["org:ygrek"]
+homepage: "https://ygrek.org/p/ocaml-extunix"
+doc: "https://ygrek.org/p/ocaml-extunix"
+bug-reports: "https://github.com/ygrek/extunix/issues"
+depends: [
+  "dune" {>= "2.2"}
+  "dune-configurator" {build}
+  "ppxlib" {>= "0.18"}
+  "ounit2" {with-test}
+  "base-bytes"
+  "base-bigarray"
+  "base-unix"
+]
```

Comment:
```suggestion
  "ocaml" {>= "4.06"}
]
```
Looks like thr c stubs do not support older versions
```
#          gcc src/time.o (exit 1)
# (cd _build/default/src && /usr/bin/gcc -std=gnu99 -O2 -fno-strict-aliasing -fwrapv -fcommon -Wall -D_FILE_OFFSET_BITS=64 -D_REENTRANT -fPIC -std=gnu99 -O2 -fno-strict-aliasing -fwrapv -fcommon -Wall -D_FILE_OFFSET_BITS=64 -D_REENTRANT -fPIC -g -I /home/opam/.opam/4.04/lib/ocaml -I /home/opam/.opam/4.04/lib/bytes -o time.o -c time.c)
# time.c: In function 'caml_extunix_asctime':
# time.c:69:3: error: unknown type name 'char_os'; did you mean 'wchar_t'?
#    char_os buf[32]; /* user-supplied buffer which should have room for
#    ^~~~~~~
#    wchar_t
# time.c:80:30: warning: passing argument 2 of 'asctime_r' from incompatible pointer type [-Wincompatible-pointer-types]
#    if (NULL == asctime_r(&tm, buf)) {
#                               ^~~
# In file included from config.h:240,
#                  from time.c:5:
# /usr/include/time.h:149:14: note: expected 'char * restrict' but argument is of type 'int *'
#  extern char *asctime_r (const struct tm *__restrict __tp,
#               ^~~~~~~~~
# In file included from config.h:20,
#                  from time.c:5:
# time.c:84:14: warning: implicit declaration of function 'caml_copy_string_of_os'; did you mean 'caml_copy_string_array'? [-Wimplicit-function-declaration]
#    CAMLreturn(caml_copy_string_of_os(buf));
#               ^~~~~~~~~~~~~~~~~~~~~~
# /home/opam/.opam/4.04/lib/ocaml/caml/memory.h:340:29: note: in definition of macro 'CAMLreturnT'
#    type caml__temp_result = (result); \
#                              ^~~~~~
# time.c:84:3: note: in expansion of macro 'CAMLreturn'
#    CAMLreturn(caml_copy_string_of_os(buf));
#    ^~~~~~~~~~
# time.c: In function 'caml_extunix_strftime':
# time.c:91:3: error: unknown type name 'char_os'; did you mean 'wchar_t'?
#    char_os buf[256];
#    ^~~~~~~
#    wchar_t
```

Reply:
thanks, applied